test/cppsuite/src/main/database.cpp

```diff
@@ -53,8 +53,24 @@ database::add_collection(scoped_session &session, uint64_t key_count)
       std::forward_as_tuple(next_id, key_count, collection_name));
     testutil_check(
       session->create(session.get(), collection_name.c_str(), _collection_create_config.c_str()));
-    _operation_tracker->save_schema_operation(
-      tracking_operation::CREATE_COLLECTION, next_id, _tsm->get_next_ts());
+    if (_operation_tracker != nullptr)
+        _operation_tracker->save_schema_operation(
+          tracking_operation::CREATE_COLLECTION, next_id, _tsm->get_next_ts());
+}
+
+void
+database::add_existing_collections(int count, int key_count)
+{
+    std::lock_guard<std::mutex> lg(_mtx);
+    testutil_assert(_next_collection_id == 0);
+    /* We don't support operation tracking across database reopen. */
+    testutil_assert(_operation_tracker == NULL || _operation_tracker->enabled() == false);
```

Comment:
you're using both `NULL` and `nullptr`. For consistency, and since it is more cpp like, can we stick to `nullptr`?